bot/cogs/watchchannels/talentpool.py

```diff
@@ -0,0 +1,276 @@
+        await LinePaginator.paginate(
+            lines,
+            ctx=ctx,
+            embed=embed,
+            empty=True,
+            max_lines=3,
+            max_size=1000
+        )
+
+    @nomination_group.command(name='unwatch', aliases=('end', ))
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def unwatch_command(self, ctx: Context, user: Union[User, proxy_user], *, reason: str) -> None:
+        """
+        Ends the active nomination of the specified user with the given reason.
+
+        Providing a `reason` is required.
+        """
+        active_nomination = await self.bot.api_client.get(
+            self.api_endpoint,
+            params=ChainMap(
+                self.api_default_params,
+                {"user__id": str(user.id)}
+            )
+        )
+
+        if not active_nomination:
+            e = Embed(
+                description=":x: **The specified user does not have an active Nomination**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        [nomination] = active_nomination
+        await self.bot.api_client.put(
+            f"{self.api_endpoint}/{nomination['id']}",
+            json={'end_reason': reason}
+        )
+        e = Embed(
+            description=f":white_check_mark: **Messages sent by {user} will no longer be relayed**",
+            color=Color.green()
+        )
+        await ctx.send(embed=e)
+        self._remove_user(user.id)
+
+    @nomination_group.group(name='edit', aliases=('e',), invoke_without_command=True)
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def nomination_edit_group(self, ctx: Context) -> None:
+        """Highlights the activity of helper nominees by relaying their messages to TalentPool."""
+
+        await ctx.invoke(self.bot.get_command("help"), "talentpool", "edit")
+
+    @nomination_edit_group.command(name='reason')
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def edit_reason_command(self, ctx: Context, nomination_id: int, *, reason: str) -> None:
+        """
+        Edits the reason/unnominate reason for the nomination with the given `id` depending on the status.
+
+        If the nomination is active, the reason for nominating the user will be edited;
+        If the nomination is no longer active, the reason for ending the nomination will be edited instead.
+        """
+        try:
+            nomination = await self.bot.api_client.get(f"{self.api_endpoint}/{nomination_id}")
+        except ClientResponseError as e:
+            if e.status == 404:
+                self.log.trace(f"Nomination API 404: Can't nomination with id {nomination_id}")
+                e = Embed(
+                    description=f":x: **Can't find a nomination with id `{nomination_id}`**",
+                    color=Color.red()
+                )
+                await ctx.send(embed=e)
```

Comment:
At this point I feel like we could benefit from an error embed utility function... Making a consistent error message has been discussed before. I don't think now is a bad time to start. You don't yet have to migrate the entire codebase over to the new function if you choose to add it now.

Reply:
I think Volcyyy prefers to use plain messages instead of embeds for these type of things. I will probably change the code back to that. I like the embeds because it stands out, but it's not necessary to use embeds for bot responses.

Reply:
I remember that it was either embeds previously or that I brought up the idea of using embeds - I agree that they look nicer, but from what I remember, it was preferred to use simple messages because embeds take up a lot of room

Reply:
I guess you're right, it does save some space.

![2019-06-29_17-10-13](https://user-images.githubusercontent.com/33516116/60386046-d02ba480-9a90-11e9-94f1-84362a2396d3.png)

Reply:
I've changed all the bot responses to plain messages, as @jchristgit requested.